.github/workflows/template-sync.yaml

```diff
@@ -22,11 +21,14 @@ jobs:
         uses: actions/checkout@v4
         # https://github.com/actions/checkout#usage
         # uncomment if you use submodules within the source repository
-        # with:
+        # comment token (and add .github path to .templatesyncignore)
+        # if you do not want actions and workflows updated
+        with:
         #   submodules: true
```

Comment:
Curious - What does `submodules` do?
Will we use it or should we remove this line for simplicity?

Reply:
> Curious - What does `submodules` do? Will we use it or should we remove this line for simplicity?

Hi @jomey . Followed the link within https://github.com/AndreasAugustin/actions-template-sync/issues/510 .

Sorry for jumping in :ninja: 
[Git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) is a git feature. Which helps to have another git repository inside of a folder within your current repository. If the target repository does not have an included git submodule (contains a `.gitmodules`  file in root) you don't need that feature. This repository most likely has not git submodules enabled, so you don't need it.

Reply:
Hi @AndreasAugustin 

Thanks for the clarification. I am aware of git submodules and mainly was wondering if that option was referred to that (without admittedly having read your docs). Having clarified that, we should remove this line in our use case.